hibernate-core/src/main/java/org/hibernate/dialect/CockroachDB1920Dialect.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.dialect;
+
+import org.hibernate.dialect.identity.CockroachDB1920IdentityColumnSupport;
+import org.hibernate.dialect.identity.IdentityColumnSupport;
+import org.hibernate.hql.spi.id.MultiTableBulkIdStrategy;
+import org.hibernate.hql.spi.id.inline.InlineIdsInClauseBulkIdStrategy;
+import org.hibernate.type.descriptor.sql.*;
+
+import java.sql.*;
+
+import java.sql.DatabaseMetaData;
```

Comment:
Seems the above import statement has been included in the previous one. Also, should we order imports by some way? I usually put static imports after non-static ones and order statements in each group by string ASC order. I might consider whether to use wildcard, though I am not sure about the convention in Hibernate on this.

Reply:
I will eliminate the duplicate.
It seems like wildcard imports are used elsewhere in Hibernate.
As far as ordering goes, the convention I was used to is to to have intra-project imports first, then imports from other projects, then static imports (with each section being alphabetical).